database/migrations/2018_09_06_150943_create_bundles_table.php

```diff
@@ -16,16 +16,21 @@ public function up()
         Schema::create('bundles', function (Blueprint $table) {
```

Comment:
Assuming you are the only instance that has run the migration before - so you are safe to update this one rather than create new migration?

Reply:
True, but yes. Me and staging.